pkg/postage/batch.go

```diff
@@ -11,14 +11,13 @@ import (
 
 // Batch represents a postage batch, a payment on the blockchain.
 type Batch struct {
-	ID            []byte   // batch ID
-	Value         *big.Int // normalised balance of the batch
-	Start         uint64   // block number the batch was created
-	Owner         []byte   // owner's ethereum address
-	Depth         uint8    // batch depth, i.e., size = 2^{depth}
-	BucketDepth   uint8    // the depth of neighbourhoods t
-	Immutable     bool     // if the batch allows adding new capacity (dilution)
-	StorageRadius uint8    // storage radius
+	ID          []byte   // batch ID
+	Value       *big.Int // normalised balance of the batch
+	Start       uint64   // block number the batch was created
+	Owner       []byte   // owner's ethereum address
+	Depth       uint8    // batch depth, i.e., size = 2^{depth}
+	BucketDepth uint8    // the depth of neighbourhoods t
+	Immutable   bool     // if the batch allows adding new capacity (dilution)
 }
 
 // MarshalBinary implements BinaryMarshaller. It will attempt to serialize the
```

Comment:
The buffer size should be now 95: `out := make([]byte, 95)`. The same change is required in tests.